src/TritonCTS/src/DbWrapper.cpp

```diff
@@ -125,7 +163,7 @@ void DbWrapper::initClock(odb::dbNet* net) {
         for (itr = iterms.begin(); itr != iterms.end(); ++itr) {
```

Comment:
range iteration is a lot easier to use and read here.
for (odb::dbITerm *iterm : iterms) 
no ugly c++ iterator necessary in c++11

Reply:
I have a change to this file that eliminates many such usages.  I've been holding it until Eder could merge his PR.